src/config-lepus.h

```diff
@@ -26,3 +26,6 @@
 
 /* MMC parameters */
 #define MMC_ID 0
+#ifdef MMC_SD
+#define MMC_ID_SD 2
```

Comment:
The other lepus boards all use mmc2, so you don't need the MMC_SD guard. Also I'd rename MMC_ID_SD to just MMC_ID2 (as it's the 2nd mmc/sd device usable by ubiboot).

Reply:
done